redisinsight/api/config/production.ts

```diff
@@ -1,8 +1,11 @@
 import { join } from 'path';
 import * as os from 'os';
+import { getUsername } from '../src/utils/get-username';
 
 const homedir = process.env.APP_FOLDER_ABSOLUTE_PATH
-  || (join(os.homedir(), process.env.APP_FOLDER_NAME || '.redisinsight-v2'));
+  || (join(
+    process['mas'] ? `/Users/${getUsername()}` : os.homedir(),
```

Comment:
does it mean that for all mac builds we will got to /Users/<user>/.ri* and this logic is needed because os.homedir for mas builds returns different path? 
Also I'm not sure that this is robust solution at all. Is it possible for user to change home dir destination? I guess yes, it is possible. If so - you might have an error that such folder does not exists or permission error

Reply:
[process.mas](https://www.electronjs.org/docs/latest/api/process#processmas-readonly) is true only for Mac Apple Store (mas), it is an undefined for mac builds
 
 yes, mas build has a different path because of [sandbox](https://developer.apple.com/documentation/security/app_sandbox). 
 Viktar wants to all mac builds should have the same folder